### DISCUSSION TITLE
Beginner Task Issue Tag?

### DISCUSSION THREAD

**Discussion by markderry** [h2oai/datatable#2690] [General] [answered]
Could we add a "Beginner" issue tag? I want to contribute  and am having a tough time identifying where would be a good issue to start with.

**Comment by oleksiyskononenko** [comment]
Yes, we had an idea to add a tag that would reflect the complexity of the issue. Something like Simple/Medium/Complex.

**Comment by st-pasha** [comment, marked as answer]
I've added the label, now all we need is to assign it to appropriate tasks.
